apis/scheme/scheme.go

```diff
@@ -14,6 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Copied from controller-runtime to avoid a dependency on specific versions
+// causing problems with anyone importing hive's api package.
+
```

Comment:
how much value do we get by keeping a copy of scheme builder vs using what non-controller-runtime dependent apis do. like openshift/api https://github.com/openshift/api/blob/master/config/v1/register.go#L11

Reply:
I'm not seeing more or less value in either, the code was so tiny and simple it seemed best to just keep doing what we're doing. We could certainly do it the way openshift/api does if that is desirable for any reason.